app/mailers/contact_mailer.rb

```diff
@@ -1,13 +1,16 @@
 # encoding: UTF-8
+require 'digest/sha2'
 class ContactMailer < ActionMailer::Base
-  default from: 'Spindelman <svarainte@fsektionen.se>', parts_order: ["text/plain", "text/html"]
-  default subject: 'Meddelande via Fsektionen.se'
+  default from: 'Spindelman <dirac@fsektionen.se>', parts_order: ['text/plain', 'text/html']
+  default subject: I18n.t('contact.message_sent_via')
+  default 'Message-ID': lambda { |v| "<#{Digest::SHA2.hexdigest(Time.zone.now.to_i.to_s)}@fsektionen.se>" }
```

Comment:
Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Line is too long. [107/100]